.github/workflows/push_snap.yml

```
@@ -0,0 +1,51 @@
+name: "Update Snap"
+
+on: [pull_request]
+# on:
+#   push:
+#     branches:
+#       - master
+#     paths-ignore:
+#       - ".github/**"
+#       - "!.github/workflows/push_aur.yml"
+#       - "**.md"
+#       - ".all-contributorsrc"
+#       - ".gitignore"
+
+jobs:
+  snap:
+    name: "Update Snap"
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'cpeditor'
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+           submodules: recursive
+           ref: snapcraft
+           fetch-depth: 0
+
+      - name: Update CMake file
+        id: update-cmake-file
+        run: |
+          git fetch --tags
+          SNAP_VERSION=$(git describe --long --tags --abbrev=8 | sed 's/-/./g' | cut -d '.' -f 1-4)
+          echo ::set-output name=SNAP_VERSION::$SNAP_VERSION
+          sed -i "/project(cpeditor LANGUAGES CXX VERSION .*/a set(SNAP_VERSION $SNAP_VERSION)" CMakeLists.txt
```

```suggestion
          sed -i "/project(cpeditor LANGUAGES CXX VERSION .*/a set(SNAP_VERSION $SNAP_VERSION)" CMakeLists.txt
```

Why not just sed `appdata.xml.in`?

---

Hmm Good idea!